os/fs/driver/mtd/smart.c

```diff
@@ -5515,7 +5515,12 @@ static int smart_journal_scan(FAR struct smart_struct_s *dev, bool print_dump)
 					fdbg("Recovery Failed : ret : %d\n", ret);
 					return ret;
 				}
-				last_recovery_seq = dev->journal_seq;
+				/* If it was valid journal, update last_recovery_seq. Otherwise, update last_checkout_seq to use next journal */
+				if (ret == OK) {
```

Comment:
Does this 'if' statement mean the following one?
				if (ret == OK) {
					last_recovery_seq = dev->journal_seq;
				} else if (ret == -EINVAL) {
					last_checkout_seq = dev->journal_seq;
				} else {
					fdbg("Recovery Failed : ret : %d\n", ret);
					return ret;
				}

Reply:
yes it is same but no advantage of line, I prepare this style, it is always easy to extend later.